### ISSUE TITLE
Open Files via Samba/OPDS

### ISSUE THREAD

**Issue by cirojr82** [pathway27/bookr-mod-vita#30]
Would be possible to implement the option to open a file using SAMBA protocol or even OPDS?!

**Comment by pathway27** [comment]
Yes, I also want this! But I need to investigate the libraries for implementation.

**Comment by cirojr82** [comment]
Great! I hope you manage to do it.

Thanks, congrats fornthe great work.

**Comment by Invictaz** [comment]
Would like to have this feature as well
